src/main/java/com/javadeobfuscator/deobfuscator/transformers/special/ParamorphismTransformer.java

```diff
@@ -101,9 +100,9 @@ public boolean canCheckEquality(JavaValue first, JavaValue second, Context conte
             }
         });
 		System.out.println("[Special] [ParamorphismTransformer] Starting");
-		if(getDeobfuscator().invaildClasses.isEmpty() && !OVERRIDE)
+		if(getDeobfuscator().invaildClasses.isEmpty())
 		{
-			System.out.println("[Special] [ParamorphismTransformer] Paramorphism not found or option not enabled. Exiting");
+			System.out.println("[Special] [ParamorphismTransformer] Paramorphism not found or option not enabled. Exiting (If you see folder class in your jar.then consider using V2 or else v1 )");
```

Comment:
What does V1 vs. V2 mean here?

Reply:
He is referring to the flags in Deobfuscator.java